geniza/corpus/annotation_export.py

```diff
@@ -111,18 +104,21 @@ def export(self, pgpids=None, modifying_users=None, commit_msg=None):
             # use PGPID for annotation directory name
             # path based on recommended uri pattern from the spec
             # {prefix}/{identifier}/list/{name}
-            doc_output_dir = os.path.join(
-                annotations_output_dir, str(document.id), "list"
+            doc_output_dir = self.document_path(document.pk)
+            doc_annotations_dir = os.path.join(
+                annotations_output_dir, doc_output_dir, "list"
             )
-            # ensure output directory exists
-            os.makedirs(doc_output_dir, exist_ok=True)
+            doc_transcription_dir = os.path.join(self.base_output_dir, doc_output_dir)
+            # ensure output directories exist
+            os.makedirs(doc_annotations_dir, exist_ok=True)
+            os.makedirs(doc_transcription_dir, exist_ok=True)
 
             # get a list of existing files so we can cleanup removed content
             # - annotation lists
-            doc_existing_files = glob.glob(os.path.join(doc_output_dir, "*.json"))
+            doc_existing_files = glob.glob(os.path.join(doc_annotations_dir, "*.json"))
```

Comment:
Glob doesn't sort, which always makes me nervous, but it doesn't seem like sorting is necessary here since it seems like `doc_existing_files` is just put into a set later on in `extra_doc_files = set(doc_existing_files) - set(doc_updated_files)`.